src/components/main/person/PersonRequestList.vue

```diff
@@ -0,0 +1,46 @@
+<script>
+  import personRequestService from 'services/person-request';
+
+  export default {
+    components: {},
+
+    data() {
+      return {
+        personRequests: []
+      };
+    },
+
+    mounted() {
+      this.getPersonRequests();
+    },
+
+    methods: {
+      getPersonRequests(query = {}) {
+        const q = Object.assign({}, query);
```

Comment:
Why copying the object? I think you can pass the query along to the `get` method